data_structures/hashing/bloom_filter.py

```diff
@@ -0,0 +1,107 @@
+"""
+See https://en.wikipedia.org/wiki/Bloom_filter
+
+The use of this data structure is to test membership in a set.
+Compared to python built-in set() it is more space-efficent.
```

Comment:
```suggestion
Compared to Python's built-in set() it is more space-efficient.
```
Python is a programming language, python is a snake.